src/applications/mhv/secure-messaging/components/shared/BlockedTriageGroupAlert.jsx

```diff
@@ -24,7 +24,7 @@ const BlockedTriageGroupAlert = props => {
   );
 
   const ehrDataByVhaId = useSelector(
-    state => state.drupalStaticData.vamcEhrData.data.ehrDataByVhaId,
+    state => state.drupalStaticData?.vamcEhrData?.data?.ehrDataByVhaId,
```

Comment:
We could technically use this approach as well
```suggestion
import { selectEhrDataByVhaId } from 'platform/site-wide/drupal-static-data/source-files/vamc-ehr/selectors';

      const ehrDataByVhaId = useSelector(selectEhrDataByVhaId);
```

Reply:
I like that suggestion, will update code.